### PR TITLE
Add Kyma version mapping for given SubAccount

### PR DESCRIPTION
**Pull request by ksputo**
**Description**

Changes proposed in this pull request:

- Added additional Kyma version mapping for SubAccount

**Related issue(s)**

### CODE REVIEW

**Review by piotrmiskiewicz — DISMISSED** [review]

File: docs/kyma-environment-broker/03-08-kyma-versions.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ type: Details
 ---
 
 Kyma Environment Broker is configured with a default Kyma version specified in the **APP_KYMA_VERSION** environment variable. This means that each Kyma Runtime provisioned by Kyma Environment Broker in a given global account is installed in the default Kyma version.
-You can also specify a different Kyma version for a global account using a ConfigMap. See the example:
+You can also specify a different Kyma version for a global account or subaccount using a ConfigMap. To specify version for a given account use following prefixes in ConfigMap keys:
+- `GA_` for global account
+- `SA_` for subaccount
+
+Subaccount version has higher priority than global account mapping - version for subaccount will always be used when provided and subaccount ID in provisioning request matches the ID in ConfigMap. See the example:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-08-kyma-versions.md:11]:
```suggestion
The subaccount version has a higher priority than the global account mapping. When provided, the version for a subaccount is used as long as the subaccount ID in the provisioning request matches the subaccount ID in the ConfigMap. See the example:
```

File: docs/kyma-environment-broker/03-08-kyma-versions.md [hunk 2]
```diff
@@ -13,10 +17,11 @@ metadata:
   name: kyma-versions
   namespace: "kcp-system"
 data:
-  3e64ebae-38b5-46a0-b1ed-9ccee153a0ae: "1.15.0-rc1"
+  GA_3e64ebae-38b5-46a0-b1ed-9ccee153a0ae: "1.15.0-rc1"
+  SA_df29c526-0c3d-4e2c-ba41-8b69cde41961: "PR-3721"
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-08-kyma-versions.md:21]: Consider adding an explanation that the value might be the version, the PR number, or `master`.

File: docs/kyma-environment-broker/03-08-kyma-versions.md [hunk 3]
```diff
@@ -13,15 +17,19 @@ metadata:
...
+  SA_4abbc9b5-055a-4571-b762-9d61b4b8a2e7: "master-1ab234"
 ```
 
-This ConfigMap contains a default version for a global account. The **3e64ebae-38b5-46a0-b1ed-9ccee153a0ae** parameter is a global account ID, and the value is the Kyma version specified for this global account.
+This ConfigMap contains a default version for a global account and a subaccount. The **3e64ebae-38b5-46a0-b1ed-9ccee153a0ae** parameter is a global account ID, and the value is the Kyma version specified for this global account. The **df29c526-0c3d-4e2c-ba41-8b69cde41961** and **4abbc9b5-055a-4571-b762-9d61b4b8a2e7** parameters are a subaccount IDs, and the values are the Kyma versions specified for those subaccounts.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-08-kyma-versions.md:25]:
```suggestion
This ConfigMap contains a default version for a global account and a subaccount. The **3e64ebae-38b5-46a0-b1ed-9ccee153a0ae** parameter is a global account ID, and the value is the Kyma version specified for this global account. The **df29c526-0c3d-4e2c-ba41-8b69cde41961** and **4abbc9b5-055a-4571-b762-9d61b4b8a2e7** parameters are subaccount IDs, and the values are the Kyma versions specified for those subaccounts. 
```

File: docs/kyma-environment-broker/03-08-kyma-versions.md [hunk 4]
```diff
@@ -13,15 +17,19 @@ metadata:
...
-  3e64ebae-38b5-46a0-b1ed-9ccee153a0ae: "1.15.0-rc1"
+  GA_3e64ebae-38b5-46a0-b1ed-9ccee153a0ae: "1.15.0-rc1"
+  SA_df29c526-0c3d-4e2c-ba41-8b69cde41961: "PR-3721"
+  SA_4abbc9b5-055a-4571-b762-9d61b4b8a2e7: "master-1ab234"
 ```
 
-This ConfigMap contains a default version for a global account. The **3e64ebae-38b5-46a0-b1ed-9ccee153a0ae** parameter is a global account ID, and the value is the Kyma version specified for this global account.
+This ConfigMap contains a default version for a global account and a subaccount. The **3e64ebae-38b5-46a0-b1ed-9ccee153a0ae** parameter is a global account ID, and the value is the Kyma version specified for this global account. The **df29c526-0c3d-4e2c-ba41-8b69cde41961** and **4abbc9b5-055a-4571-b762-9d61b4b8a2e7** parameters are a subaccount IDs, and the values are the Kyma versions specified for those subaccounts. 
+
+The Kyma version value could be either a Github tag (e.g. `1.15.0-rc1` or `1.16.0`), a version built from a pull request (e.g. `PR-3721`) or a version from `master` branch with example commit hash `1ab234`.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-08-kyma-versions.md:27]:
```suggestion
The value of the Kyma version can be a GitHub tag (for example:`1.15.0-rc1` or `1.16.0`), a version built from a pull request (for example: `PR-3721`), or a version from the `master` branch with a commit hash (for example: `master-1ab234`).
```

**Review by piotrmiskiewicz — DISMISSED** [review]

**Review by klaudiagrz — DISMISSED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**